code/game/jobs/job/research.dm

```diff
@@ -149,15 +149,65 @@
 	)
 
 	skill_points = 20
-	roleplay_difficulty = "Easy - Medium"
+	roleplay_difficulty = "Medium - Hard"
 	mechanical_difficulty = "Medium"
 
 /datum/job/seniormentalist/equip(mob/living/carbon/human/H)
-	if (H.mind?.role_alt_title == "Senior Mentalist") //Basic level of Psionics, with ability to read others.
-		psi_faculties = list("[PSI_REDACTION]" = PSI_RANK_OPERANT)
-		psi_faculties = list("[PSI_COERCION]" = PSI_RANK_MASTER)
-		psi_faculties = list("[PSI_PSYCHOKINESIS]" = PSI_RANK_OPERANT)
-		psi_faculties = list("[PSI_ENERGISTICS]" = PSI_RANK_OPERANT)
+	if (H.mind?.role_alt_title == "Senior Mentalist") //Higher level of Psionics, as Seniormost expert on them in the site.
+		psi_faculties = list(
+			"[PSI_COERCION]" = PSI_RANK_MASTER,
+			"[PSI_PSYCHOKINESIS]" = PSI_RANK_OPERANT,
+			"[PSI_REDACTION]" = PSI_RANK_OPERANT,
+			"[PSI_ENERGISTICS]" = PSI_RANK_OPERANT
+			)
+	return ..()
+
+/datum/job/mentalist
+	title = "Psychotronics Researcher"
+	department = "Science"
+	department_flag = SCI
+	selection_color = "#633d63"
+	total_positions = 2
+	spawn_positions = 2
+	supervisors = "the Senior Psychotronics Researcher, Research Director, and Assistant Research Director"
+	economic_power = 4
+	requirements = list(EXP_TYPE_SCIENCE = 480)
```

Comment:
upgrade the requirements for the senior psychotronics researcher. they are the same as this one.

Reply:
Updated to 620.